keps/sig-node/2403-pod-resources-allocatable-resources/README.md

```diff
@@ -64,6 +74,10 @@ compute device allocation, thus, alongside the existing pod resources API endpoi
 
 * Enable node monitoring agents to know the allocatable compute resources on a node, thus properly calculate the node compute resource utilization.
 
+### Non-Goals
+
+TBD
```

Comment:
We can't / shouldn't graduate to stable with a TBD list of non goals. Maybe “None”?

Reply:
yep, my fault. Planned to fix into a subsequent commit (I should have used a draft PR or marked WIP). Fixed in the next upload.